src/obfl-specification.html

```diff
@@ -1543,7 +1593,7 @@ <h4 id="L2277" class="include">td</h4>
         and <a href="#padding-atts">Padding</a> attributes groups.</p>
     </dd>
   <dt>Content Model</dt>
-  <dd>text(), leader, marker, br, evaluate, page-number, marker-reference, span,
+  <dd>text(), leader, marker, br, evaluate, page-number, marker-reference, span, external-reference,
```

Comment:
You are inconsistent in the order of the block attributes.
Everywhere above this line the `external-reference` follows the `page-number` element and here it follows the `span` element. To increase readability and to show that all these block attributes are identical everywhere, please make sure that the order of the block attributes is the same accross the whole document.

Reply:
Re-ordered attributes.